legal/satzung.md

```diff
@@ -0,0 +1,135 @@
+# Satzung
+
+
+## Name, Sitz, Geschäftsjahr
+
+1.   Der Verein trägt den Namen *open energy modelling foundation* (oemof).
+      Der Verein soll ins Vereinsregister eingetragen werden.
+      Nach der Eintragung wird der Name um den Zusatz "e.V." ergänzt.
+2.   Sitz des Vereins ist Berlin.
+3.   Das Geschäftsjahr ist das Kalenderjahr.
+    Das Gründungsjahr ist ein Rumpfjahr.
+
+
+## Zweck und Ziele
+
+1. Der Verein verfolgt ausschließlich und unmittelbar gemeinnützige Zwecke im Sinne des Abschnitts „Steuerbegünstigte Zwecke“ der Abgabenordnung.
+2. Zweck des Vereins ist die Förderung von Wissenschaft und Forschung.
+    Dieser Zweck wird verwirklicht durch
+    1. Bereitstellung wissenschaftlicher Software,
+         insbesondere von Software des "Open Energy Modelling Frameworks",
+         unter OSI-anerkannten Open-Source-Lizenzen,
+    2. Unterstützung bei der Weiterentwicklung, Wartung und Dokumentation ebendieser Software, und
+    3. $(ZWECK)
+    4. 
+
+
+## Selbstlosigkeit
+
+1. Der Verein ist selbstlos tätig. Er verfolgt nicht in erster Linie eigenwirtschaftliche Zwecke.
+2. Die Mittel des Vereins dürfen nur für seine satzungsmäßigen Zwecke verwendet werden.
+    Die Mitglieder dürfen keine Gewinnanteile und in ihrer Eigenschaft als Mitglieder auch keine sonstigen Zuwendungen aus Mitteln des Vereins erhalten.
+3. Es darf keine Person durch Ausgaben, die dem Zweck des Vereins fremd sind, oder durch unverhältnismäßig hohe Vergünstigungen begünstigt werden.
+
+
+## Organe des Vereins
+
+1. Die Organe des Vereins sind
+    1. die Mitgliederversammlung und
+    2. der Vorstand.
+2. Der Vorstand besteht aus drei Mitgliedern,
+    1. der Sprecher*in,
+    2. der Schriftführer*in und
+    3. der Kassenwart*in.
```

Comment:
I recommend to change this and broaden "the board":
* 1. Vorsitzende*r
* 2. Vorsitzende*r
* Schritführer*in
* Kassenwart*in
* Kassenprüfer*in

I think this is quite common for e.V.s. Besides, it would probably lower the hurdle to get active since the work is not only put onto few people.